.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+
+on:
+  push:
+    branches-ignore:
+      - 'dependabot/**'
```

Comment:
Will we still run tests for Dependabot PRs with this?

Reply:
Yup, since depndabot always opens a PR, this just skips the branches.